spyder/plugins/variableexplorer/widgets/main_widget.py

```diff
@@ -94,9 +100,16 @@ class VariableExplorerWidget(ShellConnectMainWidget):
     INITIAL_FREE_MEMORY_TIME_TRIGGER = 60 * 1000  # ms
     SECONDARY_FREE_MEMORY_TIME_TRIGGER = 180 * 1000  # ms
 
+    sig_open_preferences_requested = Signal()
+    """
+    Signal to open the variable explorer preferences.
+    """
+
     def __init__(self, name=None, plugin=None, parent=None):
         super().__init__(name, plugin, parent)
 
+        self.filter_on = False
```

Comment:
```suggestion
        self.filter_on = True
```

This should be True by default, as I mentioned above.